.ci/generate_requirements_bears.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import os
+import sys
+import itertools
+
+from pyprint.NullPrinter import NullPrinter
+
+from coalib.bears.BEAR_KIND import BEAR_KIND
+from coalib.collecting.Collectors import collect_bears
+
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+PROJECT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+
+PROJECT_BEAR_DIR = os.path.abspath(os.path.join(PROJECT_DIR, 'bears'))
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description='This program generates a pip requirement file for '
+                    'installation of linters that are used by the bears.')
+    parser.add_argument('--output', '-o',
+                        help='name of file to generate, or - for stdout',
+                        default=os.path.join(PROJECT_DIR,
+                                             'bears-requirements.txt'))
```

Comment:
can you change the default to `bear-requirements.txt` so it only has one plural in the filename.
And update the commit message to be the same.